Subway/Subway/Program.cs

```diff
@@ -1,16 +1,18 @@
 ﻿using System;
-using System.Collections.Generic;
 
 namespace Subway
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Subway subway = new Subway("D:\\Subway\\Subway\\Subway\\StationList.txt");
-            string path = subway.GetPath("AjaxRapids", "GoF Gardens");
+            Subway subway = new Subway(Loader.Load("D:\\Subway\\Subway\\Subway\\StationList.txt"));
+            var path = subway.GetPath("AjaxRapids", "GoF Gardens");
 
-            Console.WriteLine(path);
+            foreach (var station in path)
+            {
+                Console.WriteLine(station.StationName);
+            }
```

Comment:
You still have printing functionality in main program. Customer wants to have some universal extensible mechanism for printing/exporting.